packages/core/src/generateKernel.js

```diff
@@ -35,6 +35,12 @@ function generateKernel (opts = {}) {
     // scuttleGlobalThis config placeholder should be set only if ordered so explicitly.
     // if not, should be left as is to be replaced by a later processor (e.g. LavaPack).
     const {scuttleGlobalThis, scuttleGlobalThisExceptions} = opts
+    if (scuttleGlobalThisExceptions) {
+      // toString regexps if there's any
+      for (let i = 0; i < scuttleGlobalThisExceptions.length; i++) {
+        scuttleGlobalThisExceptions[i] = String(scuttleGlobalThisExceptions[i])
```

Comment:
I forgot why, but someone told me 
```
a = `${thingToTurnToString}` 
```
is better. I never verified and don't remember the reasoning though. :see_no_evil:

Reply:
Guess we'll never know 😱😱😱😱